packages/api/src/entities/Contribution.ts

```diff
@@ -10,6 +11,9 @@ export class Contribution extends Node<Contribution> {
   @Property({ columnType: 'text' })
   nodeID: string;
 
+  @ManyToOne(() => User)
+  author!: User;
```

Comment:
This shouldn't be necessary (and removing it shouldn't trigger a migration, but please check anyways)
```suggestion
  author: User;
```